specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/capacityReservation.json

```diff
@@ -802,6 +802,17 @@
         "sharingProfile": {
           "$ref": "#/definitions/ResourceSharingProfile",
           "description": "Specifies the settings to enable sharing across subscriptions for the capacity reservation group resource. Pls. keep in mind the capacity reservation group resource generally can be shared across subscriptions belonging to a single azure AAD tenant or cross AAD tenant if there is a trust relationship established between the AAD tenants. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details."
+        },
+        "reservationType": {
+          "type": "string",
+          "enum": [
+            "Block"
```

Comment:
Any reason we are not adding Targeted enum value too.

Reply:
You need to update the CapacityReservation sku description in this spec file (line 1003 and 1032) as to how now users can query the List Sku api to determine the CapacityReservation Types supported and alsio provide the context for capacity values restricted in case of Block Reservation type.